### ISSUE TITLE
Shib not matching organization

### ISSUE THREAD

**Issue by stephaniesimms** [CDLUC3/dmptool#116]
This problem appears to only affect new account creation via Shib.

Still weird – not associated with an organization. I have attached 2 screen shots.
There was no organization (screenshot 1). Doug selected “UVa” from the filtered list.
But looking at his profile (screenshot 2), he is still not associated with an institution.

Tried another colleague, Lorrie, who successfully logged in (UVa Organization) last week. But not today. 1st thing we noticed is the last name is her middle initial. Email is correct but no organization is matched. I have attached her output from the shib link that Brian suggested on Friday:
https://dmptool-stg.cdlib.org/cgi-bin/printenv (screenshot 3)

![file2](https://user-images.githubusercontent.com/12703638/36753584-66c17fc4-1bbb-11e8-906b-6915e149784f.jpeg)

![file4](https://user-images.githubusercontent.com/12703638/36753587-6a21d8da-1bbb-11e8-8a01-13e84d366e32.jpeg)

![screen shot 2018-02-27 at 12 31 22 pm](https://user-images.githubusercontent.com/12703638/36753153-34f99702-1bba-11e8-9832-a67198cb4525.png)

**Comment by briri** [comment]
This should be working now. I updated the logic to grab the first and last word when all we a supplied with is the displayName (e.g. 'John A. Doe') so that it results in 'John' and 'Doe'. 

There was also a bug in the logic that attempts to identify the organization. This has been updated and seems to be working for UCOP. Will wait to hear back from Sherry to determine whether or not the issue has been resolved.
